cabin-sorting/api/sortingData.js

```diff
@@ -0,0 +1,96 @@
+import path from "path";
+import fs from "fs";
+import { client, connectToDatabase } from "./mongodb";
+
+let validatedHackers;
+const validHackers = [];
```

Comment:
comment only to describe complex lines or unclear stuff in funcs. clean up extraneous ones please